attestation-service/verifier/src/lib.rs

```diff
@@ -89,12 +89,28 @@ pub type TeeEvidenceParsedClaim = serde_json::Value;
 
 #[async_trait]
 pub trait Verifier {
-    /// Verify the hardware signature and report data in TEE quote.
-    /// If the verification is successful, a key-value pairs map of TCB status will be returned,
-    /// The policy engine of AS will carry out the verification of TCB status.
+    /// Verify the hardware signature.
+    ///
```

Comment:
Can you add a comment about the `evidence` field, saying that it is the full hw evidence coming from the guest i.e. an attestation report or quote.

Reply:
Nice suggestion. Let me add some.